SPECS/python-tensorflow-estimator/python-tensorflow-estimator.spec

```diff
@@ -0,0 +1,65 @@
+%global pypi_name tensorflow-estimator
+%global _description %{expand:
+A high-level TensorFlow API that greatly simplifies machine learning programming}
+%define _enable_debug_package 0
+%global debug_package %{nil}
+
+Summary:        A high-level TensorFlow API that greatly simplifies machine learning programming
+Name:           python-%{pypi_name}
+Version:        2.11.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/tensorflow/estimator
+Source0:        https://github.com/tensorflow/estimator/archive/refs/tags/v%{version}.tar.gz#/%{pypi_name}-%{version}.tar.gz
+Source1:        %{name}-%{version}-cache.tar.gz
```

Comment:
We have the cache source, but I don't see it being unpacked anywhere in the spec.